### PR TITLE
upgrade to elastic search 7

### PR DESCRIPTION
**Pull request by depsiatwal**

### CODE REVIEW

**Review by ejayesh — APPROVED** [review]
lgtm

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ machine_python3_postgres: &machine_python3_postgres_redis
         POSTGRES_PASSWORD: debug
         POSTGRES_DB: directory_ch_search_debug
     - image: circleci/redis:3.2-alpine
-    - image: docker.elastic.co/elasticsearch/elasticsearch:6.7.1
+    - image: docker.elastic.co/elasticsearch/elasticsearch:7.1.1
```
Comment (agiamas) [inline .circleci/config.yml:13]: can we use 7.9.3 for alignment with latest stable across repos?
https://github.com/uktrade/export-opportunities/blob/706b2da7b524ab07d8303592995585b8fa957292/.circleci/config.yml#L50

File: makefile [hunk 2]
```diff
@@ -11,7 +11,7 @@ manage:
 	ENV_FILES='secrets-do-not-commit,dev' ./manage.py $(ARGUMENTS)
 
 webserver:
-	ENV_FILES='secrets-do-not-commit,dev' python manage.py runserver 0.0.0.0:8000 $(ARGUMENTS)
+	ENV_FILES='secrets-do-not-commit,dev' python manage.py runserver 0.0.0.0:8012 $(ARGUMENTS)
```
Comment (agiamas) [inline makefile:14]: dont we rely on CH search being on a certain known port for other services in dev?

**Review by agiamas — CHANGES_REQUESTED** [review]

File: makefile (hunk 2 above)
Comment (depsiatwal) [reply makefile]: good catch I had to change it locally and pushed by accident reverted

**Review by agiamas — APPROVED** [review]
lgtm

**Pull request merged**